index.html

```diff
@@ -11,9 +12,92 @@
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
+  <meta name="viewport" content="width=device-width, initial-scale=1">
 </head>
 
-<body>
-  <h1>My Resume</h1>
+<link href="https://fonts.googleapis.com/css?family=Nunito|Nunito+Sans:200" rel="stylesheet">
+
+<body class="container">
+  <header>
+    <nav>
+      <a href="#">Home</a>
+      <a href="#">About</a>
+      <a href="#">Portfolio</a>
+      <a href="#">Contact</a>
+    </nav>
+  </header>
+  <section class="CTA">
+    <img src="/images/pic01.png" alt="Here's Jonny!">
+    <h1>Jonny Appleseed - Web Dev</h1>
+    <h3>Technology is my jam!</h3>
+    <p>Occasionally I have time to work on side projects, if you're interested, let me know by signing up on this
+      website.</p>
+
+  </section>
+
+  <div class="middle-section">
+    <div class="middle-left">
+      <h2>Listen Up...</h2>
+      <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Cumque, dicta eum? Dolor aut aspernatur labore
+        dolorum tenetur, amet cupiditate eveniet ipsa, molestiae modi magnam temporibus blanditiis! Eaque odio modi
+        harum?</p>
+      <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Cumque, dicta eum? Dolor aut aspernatur labore
+        dolorum tenetur, amet cupiditate eveniet ipsa, molestiae modi magnam temporibus blanditiis! Eaque odio modi
+        harum?</p>
+    </div>
+    <div class="middle-right">
+      <h2>Stuff I'm Good At:</h2>
+      <ul>
+        <li>Disc Golf</li>
+        <li>Coding up UI</li>
+        <li>Playing lots of video games</li>
+        <li>Using bootstrap grid</li>
+        <li>Taking long naps</li>
+      </ul>
+    </div>
+  </div>
+
+  <div class="work-history">
+    <h2>Work History</h2>
+    <table>
+      <tr>
+        <th>Yrs</th>
+        <th>Company</th>
+        <th>Title</th>
+        <th>Tech Used</th>
+      </tr>
+      <tr>
+        <td>1</td>
+        <td>Apple</td>
+        <td>Software Engineer</td>
+        <td>Swift</td>
+      </tr>
+      <tr>
+        <td>2</td>
+        <td>Microsoft</td>
+        <td>Computer Scientist</td>
+        <td>.NET</td>
+      </tr>
+      <tr>
+        <td>3</td>
+        <td>Amazon</td>
+        <td>Dev Ops</td>
+        <td>AWS</td>
+      </tr>
+    </table>
+  </div>
+
+  <div class="contact">
+    <h2>Contact Me!</h2>
+    123 Main Street <br>
+    San Francisco, CA <br>
+    <br>
+    (415) 555-1212<br>
+  </div>
+  <footer>
+    Copyright Jonny Appleseed 2019
+  </footer>
+
 </body>
+
```

Comment:
Your HTML looks awesome just try and be consistent with spacing section, divs, ect